project/src/main/webapp/searchMessages.js

```diff
@@ -1,10 +1,11 @@
 function searchMessages() {
-    const query = getQuery();
-    if (query.length == 0) {
-        console.log("Please enter a question!");
+    const gmail = new GmailAPI();
+    gmail.getQuery();
+    if (gmail.question == 0) {
```

Comment:
Since question is a string, it's better to compare `gmail.question === ''`

Reply:
done!